aten/src/ATen/native/cpu/TensorCompareKernel.cpp

```diff
@@ -221,5 +236,6 @@ REGISTER_DISPATCH(_aminmax_stub, &_aminmax_kernel_impl);
 REGISTER_DISPATCH(where_kernel, &where_kernel_impl);
 REGISTER_DISPATCH(isposinf_stub, &isposinf_kernel_impl);
 REGISTER_DISPATCH(isneginf_stub, &isneginf_kernel_impl);
+REGISTER_DISPATCH(isin_default_stub, &isin_default_cpu_kernel);
```

Comment:
Actually, we don't need to use REGISTER_DISPATCH since this kernel won't benefit much from putting code in 'cpu' folder in my opinion. The main reason to put code into cpu folder is to take advantage of Intel instruction set like AVX and AVX2, and here, we mainly hope to vectorize the code. But for our case, this code won't be automatically vectorized. If you want to make this vectorized, then you have to copy `element_val` into multiple registers and then compare with test_element_data array, the cost here might be bigger than gain.
All the code that put into cpu folder will be compiled 3 times which will make the whole pytorch binary bigger as well.(1. without AVX and AVX2. 2. with AVX but no AVX2. 3. with AVX and AVX2)

Reply:
Removed the dispatch, which required a bit of a rewrite since `cpu_kernel` isn't available anymore.